apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -72,8 +72,8 @@ export default function SignUpForm() {
             .split(',')
             .map((skill) => ({ skill: skill })),
           role: router.query.position as string,
-          project: router.query.project as string,
-          id: router.query.slug as string,
+          project: router.query.slug as string,
+          //id: router.query.slug as string,
```

Comment:
@VictorDiniz89 I think we can delete this line since it's not used anymore.

Reply:
Oh yep, I'll do that